src/components/vertical-divider/vertical-divider.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
+import { StyledVerticalWrapper, StyledDivider } from './vertical-divider.style';
+
+const VerticalDivider = ({
+  h,
+  displayInline = false,
+  color = 80,
+  ...props
+}) => (
+  <StyledVerticalWrapper
+    data-component='vertical-divider'
+    pt={ props.pl || 3 }
+    pb={ props.pl || 3 }
+    pr={ props.pr || 3 }
+    pl={ props.pl || 3 }
+    height={ h }
+    displayInline={ displayInline }
+    { ...props }
+  >
+    <StyledDivider color={ color } />
+  </StyledVerticalWrapper>
+);
+
+VerticalDivider.propTypes = {
+  ...propTypes.space,
```

Comment:
I've forgotten to do this on some of them myself, but could you add this comment above this line please? `/** Styled system spacing props */` - just so anyone new looking in the future can immediately tell what this is